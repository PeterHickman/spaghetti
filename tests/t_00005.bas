10 REM Check division which is not commutative
20 let r = 4 / 2
30 if r = 2 then 100
40 print "Failed, expected 2 got" r
50 end
100 print "Ok"
