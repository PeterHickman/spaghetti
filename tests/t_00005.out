9
8
0.25
1
